### ISSUE TITLE
您好，请问我在替换为自己的数据的时候出现数据读取为0，train_path可以获取但是train_data为0，是咋回事

### ISSUE THREAD

**Issue by handsomeTeng** [Determined22/zh-NER-TF#105]
（我自己print出来的train_path.txt文件）
.\data_path\train_data.txt
（自己print train_data.txt 文件为空）
[]
Namespace(CRF=True, batch_size=64, clip=5.0, demo_model='1521112368', dropout=0.5, embedding_dim=300, epoch=5, hidden_dim=300, lr=0.001, mode='train', optimizer='Adam', pretrain_embedding='random', shuffle=True, test_data='data_path
', train_data='data_path', update_embedding=True)
C:\Users\le\AppData\Local\Programs\Python\Python36\lib\site-packages\tensorflow\python\ops\gradients_impl.py:112: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of me
mory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
train data: 0     （train_data值为0）

**Comment by handsomeTeng** [comment]
估计是你数据格式不对

**Comment by tongbao96** [comment]
你的解决了吗 我的也是0
